libkpod/container_server.go

```diff
@@ -590,6 +597,46 @@ func (c *ContainerServer) ListContainers() []*oci.Container {
 	return c.state.containers.List()
 }
 
+// ListRunningContainers returns a list of all containers that are currently running
+func (c *ContainerServer) ListRunningContainers() []*oci.Container {
```

Comment:
Is there any reason you can't just put the slice of IDs or names into closure and pass that as a filter? I see no reason why a function can't check ctr.ID() and ctr.Name() against a list included in the closure.